### ISSUE TITLE
Fix `.commands`

### ISSUE THREAD

**Issue by wei2912** [apertium/phenny#448]
`.commands` does not show the full listing of commands in every module. This is likely due to the inconsistent definition of commands in modules. For example, [modules/gci.py](https://github.com/apertium/phenny/blob/master/modules/gci.py) defines `.gci` in this manner:

```python
linking.rule = r'\.gci(?:\s+(.+))'
```

which causes the command to not be recognised by begiak. The definition of a command should be standardised to follow [modules/away.py](https://github.com/apertium/phenny/blob/master/modules/away.py) in this manner:

```python
away.commands = ['away']
away.example = '.away eating pie'
away.priority = 'low'
away.thread = False
```

**Comment by Androbin** [comment]
In `bot.py` we recognize all variants and store all commands per module.
The "commands" variant is much less powerful than the "rule" variant.
And `.command` uses the data curated by `bot.py` anyway.

The issue arises from a much simpler fact:
`.command` lists only commands with a Python Docstring.

In `bot.py`
```
if func.__doc__:
    self.doc[func.name] = (func.__doc__, example)
```

In `info.py`
```
commands = ', '.join(sorted(phenny.doc.keys()))
```

Let's instead use `phenny.commands` which holds all known commands.

**Comment by Androbin** [comment]
Fixed in a565a43b3677c08b5522008f48f684020850ee00

**Comment by Androbin** [comment]
Now let's hope we don't run into this:
![a2cce0c97ddee30-a-nw-p](https://user-images.githubusercontent.com/16437156/47711229-04667100-dc35-11e8-8465-2bd3c6ad1d69.png)
